### PR TITLE
Update descheduler CI for k8s 1.20

### PR DESCRIPTION
**Pull request by damemi**
This will depend on the descheduler 1.20 release being cut, and the kind 1.20 image being available, but getting a start on this now

### CODE REVIEW

**Review by damemi — COMMENTED** [review]
/hold
@seanmalloy @ingvagabund could you ptal to make sure I made these changes right?

**Comment by chases2** [comment]
This bump seems fine from a config perspective, but I'm not really involved with this sig.
/assign @seanmalloy 
/assign @ingvagabund 
/unassign

**Review by seanmalloy — APPROVED** [review]

**Comment by seanmalloy** [comment]
Looks good to me. @ingvagabund will need to add the /lgtm as I'm not an org members.

**Comment by ingvagabund** [comment]
Looks good
/lgtm

**Comment by damemi** [comment]
I'm actually wondering now if we should still keep the 1.17 branch jobs instead of changing that to 1.20. No reason to get rid of those for that branch. I'm going to push an update to keep them in and will need another tag @seanmalloy

**Comment by seanmalloy** [comment]
All changes look good to me. I see the kind node image has been pushed to docker hub, so should be good to merge this now.

https://hub.docker.com/layers/kindest/node/v1.20.0/images/sha256-b40ecf8bcb188f6a0d0f5d406089c48588b75edc112c6f635d26be5de1c89040?context=explore

**Comment by damemi** [comment]
Awesome, thanks for checking that
/hold cancel

**Review by soltysh — APPROVED** [review]
/lgtm

**Pull request merged**
